### DISCUSSION TITLE
[PostgreSQL] How to see column comments in Table DDL

### DISCUSSION THREAD

**Discussion by plotn** [dbeaver/dbeaver#9416] [Q&A] [answered]
- Operating system (distribution) and version: win10
- DBeaver version: 7.1.3 CE
- Additional extensions: No

- Database name and version: Postgres12

#### Describe the problem you're observing:
as shown...
![comments1](https://user-images.githubusercontent.com/11477081/89028769-aa2c9f80-d335-11ea-84a0-669d8dbe0900.png)
![comments2](https://user-images.githubusercontent.com/11477081/89028772-ab5dcc80-d335-11ea-8665-aa8ba8aecd84.png)

**Comment by uslss** [comment, marked as answer]
Hi @plotn ,
you could use "Show comments" button at the bottom toolbar
![image](https://user-images.githubusercontent.com/31996417/89036763-ee27a080-d345-11ea-8675-8a7f5d3d77f5.png)

**Comment by plotn** [comment]
sorry )))
